### ISSUE TITLE
Breadcrumb design tweak

### ISSUE THREAD

**Issue by NickHaggerty1** [equinor/energyvision#1639]
We need to adjust the final crumb text colour from Slate blue 80% to slate blue 90% (#505F6B) as we have noticed the 80% fails WCAG at that font size.

**Comment by NickHaggerty1** [comment]
@meols @SvSven

**Comment by SvSven** [comment]
@NickHaggerty1 

With slate-blue-90 the difference is difficult to see:

![image.png](https://images.zenhubusercontent.com/5dba9d22e2edb600015fe6c0/05dba081-4e63-49aa-99ec-7a131928af89)

But what we can maybe do to make it clearer is add an underline for the breadcrumb parts that are actual links, which would make it consistent with links in the text blocks:

![image.png](https://images.zenhubusercontent.com/5dba9d22e2edb600015fe6c0/4ac2dce6-8edf-485e-9228-d0da81bd2362)

To break it up a bit we can then also tweak the little chevron dividers and a slight offset on the underline:

![image.png](https://images.zenhubusercontent.com/5dba9d22e2edb600015fe6c0/2fb5852e-784c-47cc-b655-3a367180db8f)

Can also introduce a slightly bolder font weight on the link parts:

![image.png](https://images.zenhubusercontent.com/5dba9d22e2edb600015fe6c0/a4f660a3-21c9-4a84-88fa-5990e82a20c8)


However maybe the breadcrumbs are getting too attention grabbing then? 🤔

**Comment by NickHaggerty1** [comment]
Hey @SvSven sorry for late reply. Was away for a couple of days last weeks. I think the best combination would be to slightly bolder only the linked pages (no underlines) and no change to the cheverons.  I think that the combination of bolder 100% blue for linked pages and 95% for the page the user is on would be enough. (some sites dont distinguish  at all)

**Comment by NickHaggerty1** [comment]
Looks good thanks @SvSven. Approved
